internal/watch/watcher_darwin.go

```diff
@@ -0,0 +1,157 @@
+package watch
+
+import (
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/windmilleng/fsevents"
+	"github.com/windmilleng/fsnotify"
+	"github.com/windmilleng/tesseract/pkg/ospath"
```

Comment:
it seems weird for this to depend on tesseract?

Reply:
yeah, for sure. dropped.